### PR TITLE
chore: drop python 3.8 and test 3.13

### PR DESCRIPTION
**Pull request by ariostas**

### CODE REVIEW

**Comment by ariostas** [comment]
@eduardo-rodrigues when you get a chance could you generate a codecov token and add it as a secret named CODECOV_TOKEN? I noticed that it's not working because since v4 it requires a token. Thanks!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -72,3 +74,14 @@ jobs:
         user: __token__
         password: ${{ secrets.pypi_password }}
       if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+
+  pass:
+    if: always()
```
Comment (jpivarski) [inline .github/workflows/ci.yml:79]: This seems redundant, but maybe it's there for emphasis...

**Review by jpivarski — APPROVED** [review]
Looks good! Feel free to merge!

**Comment by ariostas** [comment]
@jpivarski could you update the required checks to be just this new `pass` job? I did that so they don't have to be updated every time a Python version is added or dropped (like how it is now done in Uproot).

(I'm not sure if this PR needs to be merged before you can do that)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jpivarski) [reply .github/workflows/ci.yml:79]: I see: there are different notions of "always." This isn't just a predicate that evaluates to true; it's overriding the usual situation in which a workflow will stop on first failure.

**Comment by jpivarski** [comment]
It doesn't: I'll do that right now.

**Comment by ariostas** [comment]
Thanks, Jim!

**Pull request merged**